libbeat/common/transport/tlscommon/config.go

```diff
@@ -68,7 +68,9 @@ func LoadTLSConfig(config *Config) (*TLSConfig, error) {
 	logFail(err)
 
 	cas, errs := LoadCertificateAuthorities(config.CAs)
-	logFail(errs...)
+	if cas == nil || len(cas.Subjects()) == 0 {
```

Comment:
this fixes the issue to the behaviour decribed there.
@urso do you think there is a use case when this is not needed?